src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -279,10 +280,8 @@ public WebDriver startFireFoxPortable(String browserPath) {
     private void launchChrome() {
         setupDrivermanager(ChromeDriverManager.getInstance());
         String browserName = BrowserType.CHROME;
-        DesiredCapabilities desiredCapabilities = DesiredCapabilities.chrome();
-        Object chromeCapability = populateBrowserSettings(desiredCapabilities, browserName);
-        desiredCapabilities.setCapability(ChromeOptions.CAPABILITY, chromeCapability);
-        driver = new ChromeDriver(desiredCapabilities);
+        ChromeOptions chromeOptions = (ChromeOptions) populateBrowserSettings(browserName);
```

Comment:
As far as I can tell, `populateBrowserSettings` is only ever called when the browser name / type is statically known. So, you could break up the method into three separate ones (and probably a fourth one for the common parts). That way, you'd get rid of the switch/case, and you'd get type safety, i.e. you can loose the explicit type cast.